### ISSUE TITLE
Poor error message when using `dotnet <file.dll>`  with runtimes only on Linux

### ISSUE THREAD

**Issue by sitepodmatt** [dotnet/core-setup#8597]
## Steps to reproduce
Using a dotnet runtime-only container run a dll/exe that doesnt exist.
dotnet <nonexistant.dll> somearg

In our case it was typo in our Dockerfile but we missed it  due to the confusing error message about SDKs

## Expected  behavior
Message that nonexistent.dll does not exist. Similar to how it works when SDKS are installed. 

## Actual behavior
  It was not possible to find any installed .NET Core SDKs
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download

## Environment data
`dotnet --info` output:
  It was not possible to find any installed .NET Core SDKs
  Did you mean to run .NET Core SDK commands? Install a .NET Core SDK from:
      https://aka.ms/dotnet-download

Host (useful for support):
  Version: 3.0.0
  Commit:  95a0a61858

.NET Core SDKs installed:
  No SDKs were found.

.NET Core runtimes installed:
  Microsoft.AspNetCore.App 3.0.0 [/usr/share/dotnet/shared/Microsoft.AspNetCore.App]
  Microsoft.NETCore.App 3.0.0 [/usr/share/dotnet/shared/Microsoft.NETCore.App]

To install additional .NET Core runtimes or SDKs:
  https://aka.ms/dotnet-download

tl;dr the dotnet cli command on runtimes presumes the issue is missing SDKs before more obvious issues..

**Comment by dagood** [comment]
Duplicate of https://github.com/dotnet/core-setup/issues/3490
